mods/cnc/chrome/mapchooser.yaml

```
@@ -17,47 +17,60 @@ Container@MAPCHOOSER_PANEL:
 			Height: 540
 			Background: panel-black
 			Children:
+				Label@ORDERBY_LABEL:
+					X: PARENT_RIGHT - 271
+					Y: 16
+					Width: 30
+					Height: 20
```

```suggestion
					Height: 31
```

The dropdown button size should match other dropdowns